cangen.c

```diff
@@ -458,10 +466,11 @@ int main(int argc, char **argv)
 			fprintf(stderr, "write: incomplete CAN frame\n");
 			return 1;
 		}
+		burst_sent_count++;
+		if (gap && burst_sent_count >= burst_count) /* gap == 0 => performance test :-] */
+			if (nanosleep(&ts, NULL)) return 1;
 
-		if (gap) /* gap == 0 => performance test :-] */
-			if (nanosleep(&ts, NULL))
-				return 1;
+		if(burst_sent_count >= burst_count) burst_sent_count = 0;
```

Comment:
Hmmm. Why do you reset the `burst_sent_count` to 0 here? Then the code will send in burst mode again?

Reply:
I want to have continuous bursts with gaps (if gap>0) between them.
So this way the program will send a burst, sleep for the gap time and then send another burst and so on.
if gap==0, the burst should no make a difference i think.

For example, the command "cangen can0 -c 3 -p 10 -g 1000 -L8" produces the following:
 (1588659386.791079)  can1  1E8   [8]  53 CC 70 09 25 E6 60 6C
 (1588659386.791158)  can1  375   [8]  1A 52 2E 42 0A F0 90 05
 (1588659386.791326)  can1  5CD   [8]  58 9E B3 61 10 39 BF 3B
 (1588659387.791262)  can1  3D1   [8]  79 27 26 3C 3A 15 78 5B
 (1588659387.791337)  can1  1BD   [8]  8F 65 ED 32 29 27 48 45
 (1588659387.791499)  can1  6E3   [8]  24 E6 F6 69 F1 62 07 23
 (1588659388.791451)  can1  458   [8]  32 8E 9B 1B 37 73 BC 1E
 (1588659388.791542)  can1  0FD   [8]  3E EB 7F 7C 62 A2 BE 57
 (1588659388.791667)  can1  396   [8]  7C 5D 49 5F 2D 54 CA 61
 (1588659389.791616)  can1  71B   [8]  4A 76 7D 11 86 9F 6F 3E
 (1588659389.791694)  can1  436   [8]  32 E8 5E 05 DA 6B E0 47
 (1588659389.791860)  can1  25B   [8]  A7 33 2C 01 F4 BD 0E 0A

Reply:
OKay. I see.

Reply:
Then, please keep the coding style, one space after `if` and the `burst_sent_count = 0;` into a new line.